README.md

```diff
@@ -116,6 +116,21 @@ this larger namespace.  But, we don't want to be so general that the spec become
    specified by the RSS \<language\> element.  If the rel="captions" attribute is present, the linked file is considered to be a closed captions file, regardless of what the
    mime type is.  In that scenario, time codes are assumed to be present in the file in some capacity.
 
+- **\<podcast:chapters/>**
+
+   Channel (optional | single)
+
+   This element lists optional `<podcast:chapter/>` item elements.
+
+- **\<podcast:chapter start="[(Normal Play Time)]" title="[(string)]" href="[(uri to content)]" type="[mime type]"/>**
```

Comment:
I feel that we could merge the idea of "rich content" and the chapters by having title be not-required. If the title attribute is not present, then this becomes a "sub chapter" which only purpose is to display the content present in the href - in this case we would also need an optional "end" attribute though (and if not present it is assumed that the chapter lasts until the next chapter, or the end of the podcast if this is the last chapter)

Reply:
I like that idea. I wonder if instead of making `title` optional, which seems to me part of the minimum info required to have a chapter object, if this `<podcast:chapter/>` element was moved outside of a `<podcast:chapters/>` list it might behave differently and instead act as "RichContent" that's applicable to the whole episode instead of a specific time stamp?